style.css

```diff
@@ -52,4 +52,50 @@ hr {
     letter-spacing: 0;
     text-align: center;
     margin: 0;
+}
+
+/*footer*/
+
+footer {
+    min-height: 510px;
+    width: auto;
+    background-color: #10367C;
+    padding-top: 120px;
+}
+
+.navigation{
+    display: flex;
+    justify-content: space-between;
+    margin: 0 272px 64px 104px;
+}
+
+nav p{
+    margin-top: 0;
+    margin-bottom: 24px;
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+}
+nav p:last-child{
```

Comment:
please keep one style for your code, when you use one space or no space between css rules - it can look a bit messy